core/controllers/outgoing_emails_test.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for sending outgoing mails. """
```

Comment:
Ditto -- reframe in terms of incoming request (ditto for class names below)

Reply:
File docstring renamed. I think the test class name below matches what it is trying to test. Lmk if I should adjust them further.